src/index.tsx

```diff
@@ -16,6 +18,7 @@ root.render(
       <Routes>
         <Route path={Paths.ROOT} element={<SignIn />} />
         <Route path={Paths.SIGN_UP} element={<SignUp />} />
+        <Route path={Paths.WELCOME} element={<Welcome />} />
       </Routes>
```

Comment:
It's a well known architecture decision to wrap all of this in a `App` component and import it here just like this: 
```suggestion
     <App />
```
WDTY? 
__NOTE__: There is a way we can make this much more dynamic with an array of objects, ping me tomorrow on slack if you want to do it.